tests/test_monitor.cpp

```diff
@@ -192,11 +192,11 @@ void test_monitor_versioned_basic (bind_function_t bind_function_,
     event = get_monitor_event_v2 (server_mon, NULL, NULL, NULL);
     //  Sometimes the server sees the client closing before it gets closed.
     if (event != ZMQ_EVENT_DISCONNECTED) {
-        TEST_ASSERT_EQUAL_INT (ZMQ_EVENT_CLOSED, event);
+        TEST_ASSERT_EQUAL_HEX64 (ZMQ_EVENT_CLOSED, event);
```

Comment:
I think this breaks 32 bit builds:

```
[  232s] tests/test_monitor.cpp:174:test_monitor_versioned_basic_tcp_ipv4:FAIL: Unity 64-bit Support Disabled
```

https://build.opensuse.org/package/live_build_log/network:messaging:zeromq:git-draft/libzmq/xUbuntu_14.04/i586

Reply:
Hm, can you try if https://github.com/sigiesec/libzmq/tree/fix-unity-64-bit solves this?

Reply:
It affects the first appveyor build as well, so if you rebase that branch and push it should tell you